lib/notification.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var service = require('./service').service;
+
+module.exports = {
+  create: function(movies, message) {
+    var twilioNumber = process.env.TWILIO_NUMBER;
```

Comment:
I added a `config.js` which takes care of showing the requested an error message on missing configuration. Thanks.